docs/tutorials/ruby-driver-sessions.txt

```diff
@@ -23,12 +23,13 @@ Please note that session objects are not thread safe. They must only be used by
 Creating a session from a ``Mongo::Client``
 ===========================================
 
-A session can be created by calling the ``start_session`` method on a client:
+A session can be created by calling the ``start_session`` method on a client and passing it a block:
```

Comment:
Can you please add a sentence like "When using the block form, the session will be automatically ended by the driver after the block executes" here.